src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -1030,6 +1028,8 @@ internal bool OnPointerUp(PointerRoutedEventArgs args, BubblingContext ctx = def
 				}
 			}
 
+			handledInManaged |= SetPressed(args, false, ctx);
```

Comment:
Well I think will break the event sequencing: The `PointerUp` has to be raise before the `Tap`